src/main/resources/com/google/api/codegen/go/main.snip

```diff
@@ -69,7 +69,7 @@
         CallOptions *{@view.callOptionsTypeName}
 
         // The metadata to be sent with each request.
```

Comment:
Might be worth being more specific with the docstring if you're reducing the scope to `x-goog-*` headers only.

Reply:
Done